packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -344,32 +342,33 @@ const themeStyleTag = computed(
       '--scalar-y-offset': `var(--scalar-custom-header-height, ${yPosition}px)`,
     }"
     @scroll.passive="debouncedScroll">
+    <div>deref:{{ !!dereferencedDocument }}</div>
     <!-- Header -->
     <div class="references-header">
       <slot
-        v-bind="referenceSlotProps"
+        v-bind="{ ...referenceSlotProps }"
```

Comment:
```suggestion
        v-bind="referenceSlotProps" 
```
was it not the same before?

Reply:
oh yeah, that’s a left over